README.md

```diff
@@ -1,37 +1,21 @@
 Bowling Challenge in Ruby
 =================
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday week
+For this challenge I followed a TDD approach to write a program that calculates the total score for a game of bowling, of a single player from a given array of pins knocked down by each roll.
```

Comment:
Good README. Nice to see. A complete README really helps the reviewer work out what's going on. Salut!